reactive/kubernetes_master.py

```diff
@@ -893,6 +898,10 @@ def set_final_status():
         hookenv.status_set('waiting', 'Waiting to apply keystone policy file.')
         return
 
+    if not is_flag_set('certificates.available'):
+        missing_certificates_notice()
+        return
```

Comment:
The "return" statement is great to avoid calling `hookenv.status_set("active"...)`. It seems like "black magic" the need of the `@when_not` decorator on the called function, though (`missing_certificates_notice`).